ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/CompactorTest.java

```diff
@@ -133,6 +133,7 @@ protected final void setup(HiveConf conf) throws Exception {
     MetastoreConf.setTimeVar(conf, MetastoreConf.ConfVars.TXN_OPENTXN_TIMEOUT, 2, TimeUnit.SECONDS);
     MetastoreConf.setBoolVar(conf, MetastoreConf.ConfVars.COMPACTOR_INITIATOR_ON, true);
     MetastoreConf.setBoolVar(conf, MetastoreConf.ConfVars.COMPACTOR_CLEANER_ON, true);
+    MetastoreConf.setBoolVar(conf, MetastoreConf.ConfVars.TXN_USE_MIN_HISTORY_WRITE_ID, false);
```

Comment:
If we set it as false, is there any test which run using the new logic?
CompactorTest is extended by a lot of other Test classes (TestCleaner, TestCleaner2, TestCleanerWithoutMinHistoryLevel etc).

Reply:
added new test